gravitee-am-ui/src/app/domain/applications/application/advanced/factors/factors.component.html

```diff
@@ -21,7 +21,6 @@
       <form (keydown.enter)="(false)" (ngSubmit)="patch()">
         <mfa-select
           [factors]="factors"
-          [applicationFactors]="application.factors"
```

Comment:
why is it removed?

Reply:
because now Im using single model and update patch body thru subscribers.